From ddef9472806ccd98541b61ab064bf9c4d517daa4 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:41 -0400
Subject: [redhat] kabi: Adding symbol proc_dointvec_minmax (kernel/sysctl.c)

Message-id: <c2385b51bf6a01a69210af0d9a426bd3374b7bf7.1410510773.git.poros@redhat.com>
Patchwork-id: 96119
O-Subject: [PATCH RHEL7.1 161/233] kabi: Adding symbol proc_dointvec_minmax (kernel/sysctl.c)
Bugzilla: 1071099
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    Read a vector of integers with min/max values.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/proc_dointvec_minmax
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/proc_dointvec_minmax
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/proc_dointvec_minmax

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/proc_dointvec_minmax b/redhat/kabi/kabi-module/kabi_ppc64/proc_dointvec_minmax
new file mode 100644
index 0000000..2a7aee3
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/proc_dointvec_minmax
@@ -0,0 +1,2 @@
+#1-
+0xced777cb proc_dointvec_minmax vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/proc_dointvec_minmax b/redhat/kabi/kabi-module/kabi_s390x/proc_dointvec_minmax
new file mode 100644
index 0000000..d5dddd2
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/proc_dointvec_minmax
@@ -0,0 +1,2 @@
+#1-
+0x2a3752a5 proc_dointvec_minmax vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/proc_dointvec_minmax b/redhat/kabi/kabi-module/kabi_x86_64/proc_dointvec_minmax
new file mode 100644
index 0000000..b2eedd1
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/proc_dointvec_minmax
@@ -0,0 +1,2 @@
+#1-
+0xd0298b8b proc_dointvec_minmax vmlinux EXPORT_SYMBOL
-- 
1.7.1